src/index.html

```diff
@@ -1,18 +1,62 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="product-card"
```

Comment:
The `class` attribute value `product-card` seems to be misplaced. According to the BEM methodology, the block class should be on the main element of the component, which seems to be the `<section>` element with `data-qa="card"`. Consider moving the class there.